packages/core/src/combinator/multicast.js

```diff
@@ -0,0 +1,98 @@
+import { append, remove, findIndex } from '@most/prelude'
+import { tryEvent, tryEnd } from '../source/tryEvent'
+
+const dispose = disposable => disposable.dispose()
+
+const emptyDisposable = {
+  dispose () {}
+}
```

Comment:
We can use [`disposeNone` from `@most/disposable`](https://github.com/mostjs/core/blob/master/packages/disposable/src/disposeNone.js).